### ISSUE TITLE
Is it possible to make the site show pages in two languages easily?

### ISSUE THREAD

**Issue by johann-petrak** [Phlow/feeling-responsive#260]
I am really not experienced with web site creation and was choosing jekyll and that theme because it looked very close to what I had in mind, so please excuse if those questions sound a bit dumb:

* if I want e.g. a german and an english version of the same static page or post, what would be the easiest way to make it easy to switch between those? Is there anything in the theme for that or would I just have to manually link from one to the other?
* if there is an English and German version of the main index page, is there some mechanism to show the right one automatically based on the user's browser settings or is this not possible with static sites?
* Some things are defined for the theme in _data/language.yml, how can I have several and define per page which one to use?

**Comment by Phlow** [comment]
No, multi-language is not possible. As far as I know is Jekyll not made for multi-language-websites. The Hugo website generator is better suited.

**Comment by johann-petrak** [comment]
Hmm I think at least for allowing the user to manually choose the language e.g. from the menu it should be possible, if the templates can make use of a "language: code" setting in the page title to get the something like _data/language_code.yml for the strings used in templates during generation. And having the alternate language version linked explicitly in the menu for each page as well. Jekyll should be able to support that, right?

**Comment by Phlow** [comment]
Maybe it's possible. But my theme doesn't support it.

**Comment by johann-petrak** [comment]
Sorry, I was mainly asking all these questions to see if you are interested in improving the theme or accept prs for adding this. Obviously, making it a theme that can be installed as a theme and not copied and modified would make that much easier. I have now found a simple way to support two languages in static pages with alternate navigation and will look into how to do the same for postings. But I got the feeling you are not very interested in this, so I will not post any more issues here.
